examples/ma_block_example.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+import logging
+from examples.common import Example, Config
+from flow_py_sdk import flow_client
+
+# -------------------------------------------------------------------------
+# Rretrieve a block by ID Class
```

Comment:
Maybe it would be better to just call these Examples:
```suggestion
# Rretrieve a block by ID Example
```

Reply:
in fact i don't think it need class nor example